controller/controller_server/config.py

```diff
@@ -13,6 +13,11 @@
 SECRET_PASSWORD_PARAMETER = "password"
 SECRET_ARRAY_PARAMETER = "management_address"
 
+VOLUME_CONTEXT_VOLUME_NAME_PARAMETER = "volume_name"
```

Comment:
why `PARAMETER`?

Reply:
does `FIELD` more appropriate?

Reply:
sure, or `KEY`, but consider simply not adding anything (e.g. `VOLUME_CONTEXT_VOLUME_NAME`).
in addition, `ARRAY` should probably be `ARRAY_ADDRESS`

Reply:
done